.gitignore

```diff
@@ -21,4 +20,9 @@ tmp
 tests/prediction_latency_test/*.csv
 storage_dir
 docker/dist/*
-.directory
\ No newline at end of file
+.directory
+
+# VisualStudioCode
+.vscode
+.vscode/*
```

Comment:
Isn't this the same as above?

Reply:
Yes, but it was moved in a separate article